homeassistant/components/prusalink/strings.json

```diff
@@ -15,15 +16,25 @@
       "not_supported": "Only PrusaLink API v2 is supported"
     }
   },
+  "issues": {
+    "firmware_5_1_required": {
+      "description": "The PrusaLink integration has been updated to utilize the latest v1 API endpoints, which require firmware version 4.7.0 or later. If you own a Prusa Mini, please make sure your printer is running firmware 5.1.0 or a more recent version, as firmware versions 4.7.x and 5.0.x are not available for this model.\n\nFollow the guide below to update your {entry_title}.\n* [Prusa Mini Firmware Update]({prusa_mini_firmware_update})\n* [Prusa MK4/XL Firmware Update]({prusa_mk4_xl_firmware_update})",
```

Comment:
Pleas also add to the description that to fix this repair issue the user needs to reload the config entry ater updating the printer

Reply:
I've added: "After you've updated your printer's firmware, make sure to reload the config entry in Home-Assistant to establish the connection again."

Feel free to re-phrase that, since I'm not a native speaker :)